orcid-scheduler-web/src/main/resources/orcid-scheduler-web-context.xml

```diff
@@ -30,7 +30,8 @@
 		<task:scheduled ref="orcidRecordIndexer" method="reindexRecordsOnSolr" fixed-delay="${org.orcid.scheduler.web.solrUpdateDelaySeconds:90}000"/>	
 		<task:scheduled ref="orcidRecordIndexer" method="reindexV3RecordsOnS3" fixed-delay="${org.orcid.scheduler.web.dumpApiV3:30}000"/>		      
 		<task:scheduled ref="publicProfileValidator" method="validatePublicRecords" cron="${org.orcid.scheduler.api.profile.validation.cronConfig:0 */10 * * * *}"/>
-		<task:scheduled ref="profileFundingManager" method="indexFundingSubTypes" cron="${org.orcid.scheduler.web.processFundingSubTypesForIndexing:15 15 * * * *}" />		
+		<task:scheduled ref="profileFundingManager" method="indexFundingSubTypes" cron="${org.orcid.scheduler.web.processFundingSubTypesForIndexing:15 15 * * * *}" />
+		<task:scheduled ref="orgLoadManager" method="loadOrgs" cron="${org.orcid.scheduler.web.orgImportsCronConfig}" />		
```

Comment:
@bobcaprice could you please put a default value here? just in case the variable is not defined, it run in a default schedule